### ISSUE TITLE
Update alert styles

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9171]
## Background

As the CMS announcement block was introduced, it has caused inconsistent sizing with system status alerts that appear at full screen. The issue does not appear all the time (users won't necessarily have all of these alerts at any given time), but it could be possible. The VAMC Facility page > edit node page below shows an example of what it could be like with current state:
![image](https://user-images.githubusercontent.com/90111970/169601466-0a35202d-6660-4766-9d05-383d3e9cd145.png)

Figma:
https://www.figma.com/file/EO0oAAViKMpwGUwMYpAM9E/DS-Templates?node-id=29%3A145549

Desired appearance:
<img width="468" alt="Screen Shot 2022-05-26 at 10 51 47 AM" src="https://user-images.githubusercontent.com/11279744/170547220-5e67fa42-5ec9-4866-83a7-5a84afab927c.png">
<img width="1049" alt="Screen Shot 2022-05-26 at 10 51 59 AM" src="https://user-images.githubusercontent.com/11279744/170547227-d8406d89-64bb-44c8-adc4-c075e0ebddaf.png">
<img width="816" alt="Screen Shot 2022-05-26 at 10 52 08 AM" src="https://user-images.githubusercontent.com/11279744/170547230-fa7e8920-cdfd-41d9-a231-ede69e9d7275.png">

## Acceptance Criteria
- [ ] CMS Announcement Block resized to match width of other alerts
- [ ] Background color removed from system status alerts, spacing adjusted
- [ ] Sitewide CMS design review for appropriate margins
- [ ] Implementation into Storybook

**Comment by tjheffner** [comment]
Updating the ACs based on a conversation with @rachel-kauff and @cjterdi. We looked at how the new Announcement Block and existing system status alerts looked together on the page in a variety of screen sizes. 

Cindy whipped up a bunch of examples here: https://www.figma.com/file/EO0oAAViKMpwGUwMYpAM9E/DS-Templates?node-id=29%3A145549

We talked through them and landed on some tweaks to both the info block and the system status alerts.
